assets/js/components/Mechanisms.vue

```diff
@@ -35,11 +37,30 @@
   text-shadow: 0 0 1px #999;
 }
 
-.legend .fm span {
-  width: 10px; height: 10px;
+
+.fm {
```

Comment:
this is bad specificity. it covers both the legend and the chart. it's true no directive below is understood by svg, but still bad practice.